platforms/ios/framework/src/TGMapView.h

```diff
@@ -0,0 +1,913 @@
+//
+//  TGMapView.h
+//  TangramMap
+//
+//  Created by Matt Blair on 7/9/18.
```

Comment:
General note: should go through here and annotate the interface with nullability annotations. Would "solve" some of the "this must not be nil" things.

Reply:
NVM saw the macro. These long files are hard to follow